_posts/de/newsletters/2025-04-04-newsletter.md

```diff
@@ -0,0 +1,763 @@
+      geeignet für Bitcoin angesehen, da sie zu komplex sind und das
+      Risiko bergen, die Fungibilität der durch sie gebundenen Coins zu
+      verringern. Dieses BIP führt eine einfache Covenant namens
+      *template* ein, die eine begrenzte Anzahl hochwertiger
+      Anwendungsfälle ermöglicht, ohne ein signifikantes Risiko einzugehen.
+      BIP119-Vorlagen ermöglichen **non-recursive** vollständig aufgezählte
+      Covenants ohne dynamischen Zustand" (Hervorhebung im Original).
+
+      Towns beschreibt ein Skript, das sowohl CTV als auch CSFS verwendet,
+      und verlinkt auf eine [Transaktion][mn recursive], die es auf dem MutinyNet
+      [Signet][topic signet] verwendet, die nur durch das Senden des gleichen
+      Betrags an die Skript selbst ausgegeben werden kann.
+      Obwohl es einige Debatten über Definitionen gab,
+      hat der Autor von CTV zuvor eine funktional identische Konstruktion
+      als rekursive Covenant beschrieben, und Optech folgte dieser Konvention in seiner
+      Zusammenfassung dieses Gesprächs
+      (siehe [Newsletter #190][news190 recursive]).
+
+      Olaoluwa Osuntokun [verteidigte][osuntokun enum] CTVs
+      Motivation in Bezug auf Skripte, die es verwenden,
+      die "vollständig aufgezählt" und "ohne dynamischen Zustand" bleiben.
+      Dies scheint ähnlich zu sein wie ein [Argument][rubin enumeration],
+      das der Autor von CTV (Jeremy Rubin) 2022 gemacht hat, als er den
+      Typ der pay-to-self-Covenant, den Towns entworfen hat, als "rekursiv,
+      aber vollständig aufgezählt" bezeichnete. Towns [entgegnete][towns enum],
+      dass die Hinzufügung von CSFS den angeblichen Vorteil der
+      vollständigen Aufzählung untergräbt. Er bat um eine Aktualisierung der
+      CTV- oder CSFS-BIPs, um Anwendungsfälle zu beschreiben, die "irgendwie
+      sowohl beunruhigend als auch durch die Kombination von CTV und CSFS verhindert werden".
+      Dies könnte in der jüngsten Aktualisierung von BIP119 geschehen sein, die eine
+      "self-reproduzierende Automaten (umgangssprachlich als SpookChains bezeichnet)"
+      beschreibt, die mit [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] möglich wären,
+      aber nicht mit CTV+CSFS.
+
+    - *Werkzeuge für CTV und CSFS:* Towns [notierte][towns ctvmot],
+      dass er es schwierig fand, bestehende Werkzeuge zu verwenden,
+      um sein rekursives Skript zu entwickeln, was auf einen Mangel
+      an Bereitschaft für die Bereitstellung hinweist. Osuntokun
+      [sagte][osuntokun enum], dass die Werkzeuge, die er verwendet,
+      "ziemlich einfach" seien. Weder Towns noch Osuntokun erwähnten,
+      welche Werkzeuge sie verwendet haben. Nadav Ivgi [stelle][ivgi minsc]
+      ein Beispiel mit seiner [Minsc][]-Sprache vor und sagte, dass er daran arbeitet,
+      Minsc zu verbessern, um solche Dinge einfacher zu machen.
+      Es unterstützt Taproot, CTV, PSBT, Deskriptoren, Miniscript, rohe Skripte,
```

Comment:
"Script" bezieht sich hier auf die Sprache, in der Bitcoins Skripte geschrieben sind:

```suggestion
      Es unterstützt Taproot, CTV, PSBT, Deskriptoren, Miniscript, pures Script,
```